app/react/Layout/DocumentsList.js

```diff
@@ -69,6 +77,9 @@ class DocumentsList extends Component {
       connectionsGroups,
       LoadMoreButton,
       rowListZoomLevel,
+      location: {
```

Comment:
please use the existing state.library.sidepanel.multiEditOpts instead of reparsing from the query.

Reply:
Also, I think thesaurusName is not usually populated, so you'll need to look up the id in state.thesauris.find(t => t._id.toString() == opts.thesaurus)